src/MSLGenerator.cpp

```diff
@@ -116,14 +116,17 @@ static const char * translate_input_semantic(MSLGenerator::Target target, const
         if (target == MSLGenerator::Target_VertexShader)
         {
             if (is_semantic(semantic, "POSITION", 0)) return "attribute(0)";
```

Comment:
I imagine every application may want to do this in a different way. Maybe we can accommodate that with a callback or specifying the mapping up front and storing it in a table?